extensions/sql-database-projects/src/dialogs/publishOptionsDialog.ts

```diff
@@ -29,7 +29,7 @@ export class PublishOptionsDialog {
 	}
 
 	protected initializeDialog(): void {
-		this.optionsTab = utils.getAzdataApi()!.window.createTab(constants.publishOptions);
+		this.optionsTab = utils.getAzdataApi()!.window.createTab(constants.publishingOptions);
```

Comment:
why is this being changed from publish to publishing? In the UX meeting, we decided to use publish

Reply:
Not an intentional changes, Ignore this here, this has been handled in CSS and UI updates PR, which will be merged next to this pr. don't know what causing this change to revert, may be merging to main causing this!

Reply:
You can do a git blame via the web UI here. Was caused by https://github.com/microsoft/azuredatastudio/commit/450160748862cceec80f21142b463b50a6736bc6#diff-a763d2aa345965b3ebe712da07632ed691cbe320ec0763fa62ccee87810d63e2R30.

Best to revert unintended changes.